src/main/java/io/github/cottonmc/resources/mixin/MixinVanillaOregen.java

```diff
@@ -13,21 +13,21 @@
 
 
     @Inject(method = "addDefaultOres", at = @At(value = "HEAD"))
```

Comment:
All three of these need to be given the `cancellable = true` flag while we're here.

Reply:
Did that.